boards/samr30-xpro/include/periph_conf.h

```diff
@@ -96,6 +96,18 @@ static const spi_conf_t spi_config[] = {
         .miso_pad = SPI_PAD_MISO_0,
         .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
         .gclk_src = SAM0_GCLK_MAIN,
+    },
+    {
```

Comment:
```suggestion
    { /* EXT1 & EXT3 */
```

So the user knows which is which.